src/app/personside/infotabs/meldinger/traadliste/TraadListeElement.tsx

```diff
@@ -59,6 +59,11 @@ const EtikettStyling = styled.div`
     margin-top: 0.2rem;
 `;
 
+const ContentStyle = styled.div`
+    flex-grow: 1;
+    width: 0px;
```

Comment:
Dette er ganske funny css, da pleier jeg å slenge på en `/* ie11-fix */` for å klargjøre